character/human/body/.code/rig/arms/ik_clavicle/ik_clavicle.py

```diff
@@ -78,21 +78,19 @@ def main():
         
         #fix pole vector
         loc = cmds.spaceLocator(n = 'clavicle_rotateLock_%s' % side)[0]
+        locXform = cmds.group(loc ,n = 'xform_clavicle_rotateLock_%s' % side)

```

Comment:
use space.create_xform_group(loc)
this will handle the xform_ naming, and if there were an offset it would automatically add it.